src/Classes/Utils/Messages.ts

```diff
@@ -0,0 +1,7 @@
+export class Messages {
+    public static deprecationMessage(oldProperty: string, newProperty: string) {
+        if (console) {
+            console.warn(`The property ${oldProperty} is obsolete and will be removed in a future release. Please use the new property ${newProperty}.`);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5139697)